decidim-core/lib/decidim/query_extensions.rb

```diff
@@ -15,7 +15,7 @@ def self.extend!(type)
           type !types[ProcessType]
           description "Lists all processes."
 
-          resolve ->(_obj, _args, ctx) {
+          resolve lambda { |_obj, _args, ctx|
```

Comment:
Maybe it's a problem on codecov's end since they are two lines but we aren't using do .. end?